### PR TITLE
Add security group rules to allow uperf tests.

### PR DESCRIPTION
**Pull request by jmencak**
This is a temporary role that allows the uperf test client to work. Once openshift-ansible supports adding custom TCP/UDP inbound rules, this role should be deleted and replaced by that functionality.

### CODE REVIEW

File: roles/openshift_on_openstack_secgroup/vars/main.yml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+---
+# The path to the OpenStack RC file on the openstack-server, may be named differently than other servers.
+openstack_rc: "{{ lookup('env', 'openstack_rc_path')|default(ansible_user_dir ~ '/overcloudrc', true) }}"
+# OpenStack security group name to add inbound rules to.
+security_group_name: 'openshift-ansible-scale-ci.example.com-common-secgrp'
+# TCP/UDP inbound port range for the uperf client.
+#uperf_range: '20000:20200'
+uperf_range: '1024:65535'
```
Comment (mbruzek) [inline roles/openshift_on_openstack_secgroup/vars/main.yml]: There has to be a better way than disabling all the ports! 

After discussing with @jmencak we agreed that trying to get all the specific ports could take forever. 

I would like to document a way to undo this change in case we want to test as customers would with the default security groups.

**Comment by jmencak** [comment]
Yeah, there is almost always a better way.  If you have any suggestions, happy to hear them.  Made the PR more configurable, you can now disable this functionality completely by setting "permissive_secgroup_enable" to false.  By default, it is enabled to make uperf (and possibly other stuff) to work.  Also made the ports and security group configurable.
